srcs/parser/token/token_globing_parse_none.c

```diff
@@ -15,6 +15,8 @@ static char	*s_expand_escape_char_not_inhibited(char *str)
 		{
 			if (str[j] == '\\' && str[j + 1] != '\\')
 				j++;
+			else if (str[j] == '\\' && str[j - 1] != '\\')
```

Comment:
at first loop, `j == 0` so that you check if `str[-1] != '\\'`, but it could result in forbidden memory access attempt